chainercv/utils/image/tile_images.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import division
+
+import math
+import numpy as np
+
+
+def tile_images(imgs, n_col, pad=2, fill=0):
+    """Make a tile of images
+
+    Args:
+        imgs (numpy.ndarray): A batch of images whose shape is BCHW.
+        n_col (int): Number of columns in a tile.
+        pad (int): Amount of pad. Default is 2.
+        fill (float, tuple or ~numpy.ndarray): The value of padded pixels.
+            If it is :class:`numpy.ndarray`,
+            its shape should be :math:`(C, 1, 1)`,
+            where :math:`C` is the number of channels of :obj:`img`.
+
+    Returns:
+        ~numpy.ndarray:
+        An image array in CHW format.
```

Comment:
How about adding a note about the output size?
`The size of this image is :math:((H + pad) * \lceil B / n_col \rceil, (W + pad) * n_col)`